src/Services/AvatarExistsLimitService.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+
+namespace WCM\User\Avatar\Services;
+
+
+class AvatarExistsLimitService implements ServiceInterface {
+
+	/** @var string */
+	private $key;
+
+
+	/**
+	 * @param string $key
+	 */
+	public function __construct( $key )
+	{
+		$this->key = $key;
+	}
+
+	/**
+	 * @param array $file
+	 * @return string
+	 */
+	public function setup( Array $file = [ ] )
+	{
+
+		$screen = filter_input( INPUT_POST, 'screen_id', FILTER_SANITIZE_STRING );
+		if ( ! in_array( $screen, [ 'profile', 'user-edit' ], TRUE ) )
+		{
+			return $file;
+		}
+
+		$user_id = filter_input( INPUT_POST, 'user_id', FILTER_SANITIZE_NUMBER_INT );
+
+		if( false === get_userdata($user_id) ){
+			$file['error'] = 'Cannot find user.';
```

Comment:
We have the [`createpot.sh`](https://github.com/wecodemore/wcm-avatar/blob/b4ecf0e12021015740958a878200c526e915ced8/create-pot.sh) bash script for translations. This means, that we can use `__()` and `_e()` and related functions.